iidm/iidm-impl/src/main/java/com/powsybl/iidm/network/impl/GeneratorAdderImpl.java

```diff
@@ -121,7 +129,7 @@ id, getName(), isFictitious(), energySource,
                                     minP, maxP,
                                     voltageRegulatorOn, regulatingTerminal != null ? regulatingTerminal : terminal,
                                     targetP, targetQ, targetV,
-                                    ratedS);
+                                    ratedS, isCondenser);
```

Comment:
@geofjamg I think that we have to add a validity checks or not, need to think about it: 
- targetP must be zero if condenser = true.
- or we consider that all active power related attributes must be ignored? For example, the generator should be excluded form active power distribution so it is natural to ignore related data.
What do you think?

Reply:
> * targetP must be zero if condenser = true.

I'd be against that, or, for a practical use, it would require `isCondenser` to be renamed to `isCondensing` + make it variant dependent.

> * or we consider that all active power related attributes must be ignored? For example, the generator should be excluded form active power distribution so it is natural to ignore related data.
What do you think?

My call would be that isCondenser should be used/checked only when targetP is zero.
Conversely, if targetP is not zero, then isCondenser should not be used/checked.